# @Exception

function foo() {
  return [2, 4, 8][3]
}

foo()
echo "bad"
